src/services/geometry/richgeo/IrtGeoDRICH.cc

```diff
@@ -143,8 +141,8 @@ void richgeo::IrtGeoDRICH::DD4hep_to_IRT() {
         sensor_info.size             = sensorSize;
         sensor_info.surface_centroid = posSensor;
         sensor_info.surface_offset   = surfaceOffset;
+        sensor_info.surface_normZdir = normZdir;
```

Comment:
I just realised `surfaceOffset` and `normZdir` are parallel... perhaps we just use `surfaceOffset.Unit()` downstream so we don't have to increase the size of `m_sensor_info` with duplicated information.